Makefile

```diff
@@ -448,6 +448,21 @@ ggml-opencl.o: ggml-opencl.cpp ggml-opencl.h
 	$(CXX) $(CXXFLAGS) -c $< -o $@
 endif # LLAMA_CLBLAST
 
+ifdef LLAMA_VULKAN
+	CFLAGS  += -DGGML_USE_VULKAN
+	CXXFLAGS  += -DGGML_USE_VULKAN
+	LDFLAGS += -lvulkan
+	OBJS    += ggml-vulkan.o
+
+ifdef LLAMA_VULKAN_CHECK_RESULTS
+	CFLAGS  += -DGGML_VULKAN_CHECK_RESULTS
+	CXXFLAGS  += -DGGML_VULKAN_CHECK_RESULTS
+endif
+
```

Comment:
I think this should use the `MK_xx` variables instead. The define should only be needed in `MK_CPPFLAGS`, since that's passed to both C and C++ code.

Reply:
Those are new, I didn't notice the Makefile changes. Should be correct now.